transitive_deps.bzl

```diff
@@ -6,4 +6,4 @@ load("@rules_rust//rust:repositories.bzl", "rust_repositories")
 def cargo_raze_transitive_deps():
     """Loads all dependnecies from repositories required for cargo-raze"""
     rules_foreign_cc_dependencies()
-    rust_repositories()
+    rust_repositories(include_rustc_srcs = True)
```

Comment:
Is both this and the `RULES_RUST_TOOLCHAIN_INCLUDE_RUSTC_SRCS` change in the bazelrc needed?

Reply:
You're right, it's necessary to do both.